ctx.go

```diff
@@ -1761,3 +1732,17 @@ func (c *DefaultCtx) Bind() *Bind {
 	}
 	return c.bind
 }
+
+// Converts a string value to a specified type, handling errors and optional default values.
+func Convert[T any](value string, convertor func(string) (T, error), defaultValue ...T) (*T, error) {
```

Comment:
Are there any reason we return pointer here?

Reply:
@dozheiny

Reply:
@efectn I made this pointer because `Convert` returns nil when reaching to error.
but you are right. I think it's better to avoid nil pointers.